README.md

```diff
@@ -78,6 +78,19 @@ export default MyComponent extends Component {
 />
 ```
 
+## Testing
+
+A test helper `fillInEditor` is exported from `ember-tui-editor/test-support/helpers`, which could be used in integration and acceptance tests:
+
+```js
+import { fillInEditor } from 'ember-tui-editor/test-support/helpers';
```

Comment:
Wondering if the export path should be maybe simplified to `ember-tui-editor/test-support`. I guess it would ease discoverability of that test helper.